packages/fluxible-addons-react/tests/unit/lib/connectToStores.js

```diff
@@ -91,62 +91,58 @@ describe('fluxible-addons-react', function () {
             done();
         });
 
-        // Decorators not supported with babel 6
-        it.skip('should get the state from the stores using decorator pattern', function (done) {
-            //@connectToStores([FooStore, BarStore], (context) => {
-            //    return {
-            //        foo: context.getStore(FooStore).getFoo(),
-            //        bar: context.getStore(BarStore).getBar()
-            //    };
-            //}) class Component extends React.Component {
-            //    static contextTypes = {
-            //        executeAction: PropTypes.func.isRequired
-            //    };
-            //
-            //    onClick() {
-            //        this.context.executeAction(function (actionContext) {
-            //            actionContext.dispatch('DOUBLE_UP');
-            //        });
-            //    }
-            //
-            //    render() {
-            //        return (
-            //            <div>
-            //                <span id="foo">{this.props.foo}</span>
-            //                <span id="bar">{this.props.bar}</span>
-            //                <button id="button"
-            //                        onClick={this.onClick.bind(this)}/>
-            //            </div>
-            //        );
-            //    }
-            //}
-            //
-            //var WrappedComponent = provideContext(Component);
-            //
-            //var container = document.createElement('div');
-            //var component = ReactDOM.render(<WrappedComponent
-            //    context={appContext}/>, container);
-            //var domNode = ReactDOM.findDOMNode(component);
-            //expect(domNode.querySelector('#foo').textContent).to.equal('bar');
-            //expect(domNode.querySelector('#bar').textContent).to.equal('baz');
-            //
-            //ReactTestUtils.Simulate.click(domNode.querySelector('#button'));
-            //
-            //expect(domNode.querySelector('#foo').textContent).to.equal('barbar');
-            //expect(domNode.querySelector('#bar').textContent).to.equal('bazbaz');
-            //
-            //expect(appContext.getStore(BarStore).listeners('change').length).to.equal(1);
-            //expect(appContext.getStore(FooStore).listeners('change').length).to.equal(1);
-            //
-            //ReactDOM.unmountComponentAtNode(container);
-            //
-            //expect(appContext.getStore(BarStore).listeners('change').length).to.equal(0);
-            //expect(appContext.getStore(FooStore).listeners('change').length).to.equal(0);
+        it('should get the state from the stores using decorator pattern', function (done) {
+            @connectToStores([FooStore, BarStore], (context) => {
```

Comment:
we should really remove support for decorators since the api is legacy and will never make it into the spec now.

Reply:
I'm glad you said that. But that would be another breaking change, right? So, I would keep it for now since this PR just updates tooling. Then, when we move to the beta versions where we really start introducing breaking changes, I would definitely removed it.